### DISCUSSION TITLE
74ALS990

### DISCUSSION THREAD

**Discussion by berger1920** [slabbi/Z80-CPU-Tester#4] [General]
Hi,

This chip is not easy to find, at least not new. For example at some of my usual suppliers:

Mouser: 0 stock.
Digikey: 0 stock.
TME: Not listed.
RS: Not listed.
Farnell: Not listed.
Reichelt: Not listed.

I got mine here: [Kessler-electronic](https://www.ebay.de/str/kesslerelectronic)

https://www.ebay.de/itm/310655532474

It is genuine, I have verified it. I have seen others for sale via eBay in any case, but at higher prices and/or suspicious.

Best,
Gerardo "Berger"

**Comment by berger1920** [comment]
Mouser has currently 95 "SN74ALS990N" in stock.

**Comment by wrljet** [comment]
I bought 20 pieces from eBay (USA) for $27 with postage.
It was a chip I'd never heard of before this project, and looks cool, so I wanted to have some in-stock.

Bill

**Comment by berger1920** [comment]
Wow! That is a very good price. I think I've found that sale on eBay. In this case, although I would have been paying higher shipping costs plus Customs fees, it would have been cheaper than what I paid here for only 2 units.  :D 

Although if Mouser had had them in stock for the past month, I would have preferred to buy from them: I've received so many counterfeit chips from eBay... I don't think that's your case though. Let us know how it goes. :)

**Comment by wrljet** [comment]
Well, it took me a while to get around to building up the PCB, and then to remember to post here,
but the chips I got (and the full project) did work.

Bill
![Z80-CPU-Tester_2025-02-28](https://github.com/user-attachments/assets/528e350c-6e4b-405f-a346-dfb2ce18397e)

**Comment by slabbi** [comment]
Thanks a lot for the feedback and the picture.
The PCB in purple looks really great.
